lib/puppet/type/concat_file.rb

```diff
@@ -58,68 +54,98 @@ def exists?
   end
 
   newparam(:order) do
-    desc "Controls the ordering of fragments. Can be set to alphabetical or numeric."
-    defaultto 'numeric'
+    desc "Controls the ordering of fragments. Can be set to :alpha or :numeric."
```

Comment:
Ah, I missed that it only emits a warning but it makes it seem as if the intent is that fragments should target concat_file by path or title.